scripts/monitoring/auto_scale_workers.py

```diff
@@ -47,15 +46,16 @@ def get_sqs_queue_by_name(queue_name):
     )
     if queue_name == "":
         raise ValueError("Queue Name cannot be empty")
-
+    queue = None
     try:
         queue = sqs.get_queue_by_name(QueueName=queue_name)
     except botocore.exceptions.ClientError as ex:
         if (
             ex.response["Error"]["Code"]
             != "AWS.SimpleQueueService.NonExistentQueue"
         ):
-            raise Exception("Cannot get queue: {}".format(queue_name))
+            # raise Exception("Cannot get queue: {}".format(queue_name))
+            print("Cannot get queue: {}".format(queue_name))
```

Comment:
Can you replace the message with `Unable to fetch {} queue details".format(queue_name)